features/step_definitions/admin_steps.rb

```diff
@@ -0,0 +1,130 @@
+Given(/^I have an admin account$/) do
+  @admin = AdminUser.create(email: "admin@example.com", password: "password", 
+    password_confirmation: "password")
+  # byebug
+end
+
+When(/^I visit the admin panel url$/) do
+  visit("/admin/login")
+end
+
+When(/^I enter my admin email$/) do
+  fill_in "Email*", with: "admin@example.com"
+end
+
+When(/^I enter my admin password$/) do
+  fill_in "Password*", with: "password"
+end
+
+When(/^I click submit$/) do
+  click_button("Login")
+end
```

Comment:
This is misleading. The test doesn't say "I click Login", it says "I click submit". Either you should change your test to say `When I click "Login"`, or you should change your button text to say "submit".